### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by Suk-RAM** [olive-editor/olive#2225] [Ideas]
Dear Olive friends,

How about a roadmap towards a final Olive 0.2? A roadmap that shows the tasks that need to be done before the release of a stable Olive 0.2. And a roadmap that estimates when each task is probably going to be finished. I think such a roadmap would give us all orientation beyond the long time present announcement on https://www.olivevideoeditor.org/index.php that Olive 0.2 will soon be available.

Regards,

Suk-RAM

**Comment by Simran-B** [comment]
The following roadmap has been shared via Patreon and Discord some time ago but I think it's still up-to-date (with minor edits):

- Continue improving the UI, particularly fixing usability issues with regard to nodes and timeline
- Implement a lot more nodes/effects so people can do more things
- Iron out performance and stability issues. A lot of crashes have been fixed, but there's always more to be done, and there are some slowdowns that are not ideal

There is no time estimate because it's unclear how much effort each task takes (it also depends on what and how many issues users report), and because of the limited development resources. Therefore, the only thing that can be said is that v0.2 will be done when it's done.

**Comment by Suk-RAM** [comment]
Hi Simran-B, 

Thanks for summing up the roadmaps from all websites that belong to Olive! 
Wouldn't it be easier to maintain just one roadmap and then refere to this roadmap from all those websites? By the way, each visitor - whatwever website he/she is visiting - can read the full roadmap.   

Regards, 

Suk-RAM
